docs/topic-guides/native-api.txt

```diff
@@ -0,0 +1,88 @@
+.. _native-api:
+
+Native API (experimental)
+=========================
+
+.. warning:
+
+    The Native API is currently an experimental API.
+
+The Native API allows developers to seamlessly move between training in local
```

Comment:
The first sentence seems a bit ambiguous to me in what the main problem Native API wants to solve. When you say "training in local Python script", do you mean writing code for training in your local machine or literally training in local Python scripts? If it's the latter, it has the same meaning as the local mode does.

What do you think of the following sentences as the first paragraph?

Native API provides a very easy-to-use API for us to easily making use of Determined's capability to train multiple models on multiple machines. It supports Tensorflow Keras and Estimator with idiomatic framework patterns. A typical use case is that before using Determined we have training code written to integrate with the native frameworks and we want to port the model to the Determined API. Using the Native API, we only need minimum effort to do this.

Reply:
(I copied this paragraph straight from the Native API tutorial)

I modified "training in local Python scripts" to "training in a local development environment" on both to make that clearer. Apart from that, I think the paragraph as it stands is clearer than the one you've proposed -- I guess it's a matter of style at the end of the day...

Reply:
Do you mean training in a local development environment with Determined or without Determined?

Reply:
without Determined

Reply:
"local development environment" implies not controlled by Determined